client/modules/IDE/components/SketchList.jsx

```diff
@@ -194,6 +212,16 @@ class SketchListRowBase extends React.Component {
           <td>{formatDateCell(sketch.updatedAt, mobile)}</td>
           {this.renderDropdown()}
         </tr>
+        {this.state.preview ? (
+          <ExamplePreview
+            url={`/editor/${this.props.username}/projects/${slugify(
+              this.props.sketch.name,
+              '_'
+            )}`}
```

Comment:
Only the p5 user has slugified project URLs.  URLs for other users are based on the id.  But I don't think the `url` is what we want to pass down to the `ExamplePreview` anyways.  If we have the entire `sketch` object (I think we do?) then we should pass that down, and then we don't need any fetching in the `ExamplePreview`.  If it's just a summary of the sketch then we would pass down the `id` and `username` of the sketch and then fetch the complete details in the `ExamplePreview`.